src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -1670,6 +1867,40 @@ private void ResponseCallback(IAsyncResult asynchronousResult)
             }
         }
 
+        private static async Task VerifyNewConnection(string responseContent, Task<Socket> secondAccept, Task<WebResponse> currentResponseTask)
+        {
+            var secondServer = await secondAccept;
```

Comment:
Nit: please only use var when the type is evident on the right-hand side (ctors, explicit casts)